### PR TITLE
Mkind 2018 02 idea remarks

### PR DESCRIPTION
**Pull request by mkind**
* [x] adding remark template to ideas
* [x] setting remark via on popover hide event
* [x] fetching remark on popover show event

**depends on** https://github.com/liqd/adhocracy4/pull/249

Still to do (maybe as issues/new tasks):
- [x] styling if rating is enabled (remarkbutton ends up in second line)
- [ ] a11y?
- [ ] error handling
- [x] submit button instead of onLeaveFocus ?

### CODE REVIEW

File: meinberlin/apps/moderatorremark/api.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from adhocracy4.api.mixins import AllowPUTAsCreateMixin
+from adhocracy4.api.permissions import IsModerator
+
+from .models import ModeratorRemark
+from .serializers import ModeratorRemarkSerializer
+
+
+class ModeratorRemarkViewSet(AllowPUTAsCreateMixin,
+                             mixins.UpdateModelMixin,
+                             viewsets.GenericViewSet):
```
Comment (kleingeist) [inline meinberlin/apps/moderatorremark/api.py]: provide a `retrieve()` method or try adding the `mixins.RetrieveModelMixin` from drf to allow for `GET`ting objects

File: meinberlin/apps/moderatorremark/models.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from adhocracy4.models.base import UserGeneratedContentModel
+from meinberlin.apps.ideas import models as idea_models
+
+
+class ModeratorRemark(UserGeneratedContentModel):
```
Comment (kleingeist) [inline meinberlin/apps/moderatorremark/models.py]: missing migration for the new model

Comment (mkind) [reply meinberlin/apps/moderatorremark/models.py]: done in 
abeee5c

**Comment by mkind** [comment]
> AttributeError 'Proposal' object has no attribute 'moderatorremark_set'

I see. It seems I need to switch to `GenericForeignKey`s to attach remarks to the different kinds of ideas.

**Comment by fuzzylogic2000** [comment]
This is no joking matter!

**Pull request merged**